main.go

```diff
@@ -7,12 +7,28 @@ import (
 	"log/slog"
 	"os"
 	"os/exec"
+	"os/signal"
 	"runtime"
 	"strconv"
 	"strings"
+	"syscall"
+	"time"
 )
 
-var logLevel = new(slog.LevelVar)
+var (
+	logLevel = new(slog.LevelVar)
+	// termWait is the time to wait before forcefully terminating the child process
+	termWait = 5 * time.Second
```

Comment:
We provide the pod 5 minutes in k8s to shut down after receiving a SIGTERM.

Pods may need to do work before they shut down, for example, wait for any processing jobs to complete before they terminate.

Should this be increased?

Reply:
The name of this is misleading, this is the timeout for SIGINT, not SIGTERM. I'll rename to something more clear

Reply:
Renamed